D2BotSoloPlay.dbj

```diff
@@ -1212,7 +1243,7 @@ function main () {
 						lrRes = Math.min(75, me.getStat(41) - resPenalty);
 						prRes = Math.min(75, me.getStat(45) - resPenalty);
 						areaName = !!me.area ? Pather.getAreaName(me.area) : "";
-						statusString = "Lvl: " + me.charlvl + " (" + myExp + "%) (Diff: " + diffName + ") (A: " + areaName + ") (G: " + myGold + ") (F: " + frRes + "/C: " + crRes + "/L: " + lrRes + "/P: " + prRes + ")";
+						statusString = "C-Name: " + me.name + " | Lvl: " + me.charlvl + " (" + myExp + "%) (Diff: " + diffName + ") (A: " + areaName + ") (G: " + myGold + ") (F: " + frRes + "/C: " + crRes + "/L: " + lrRes + "/P: " + prRes + ")";
```

Comment:
I don't think C-Name needs to be there. Just starting out with the character name seems fine

Reply:
I'll remove it